+

```diff
@@ -0,0 +1,6 @@
+Merge branch 'master' into feat-codesandboxes-description
```

Comment:
Accidental add?

Reply:
why? there were some modifications to the docs so I needed to fix merge issues locally so merged master into the branch.

Reply:
Actually it is a file named `+`  and staged 😁 

https://github.com/nuxt/nuxtjs.org/blob/a8b4d84b3c01731a6da064e731468da1616a6b15/%2B

Reply:
wow. how did that get there. very weird. Thanks. Removed